README.md

```diff
@@ -139,28 +176,25 @@ The function allows to extract step datas to save in the browser storage and per
 
 ### Manager
 
-Now steps are created, we will create the manager to manage all these steps.
+Now the steps are created, we will create the Manager to manage all these steps.
```

Comment:
_Now that the steps are created, we will create the Manager to manage them._

Reply:
C'est corrigé